_posts/2021-10-01-automated-testing-for-opensearch-releases.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "Automated testing for OpenSearch releases"
+authors: 
+  - setiah
+date: 2021-10-01
+categories: 
+  - technical-post
+twittercard:
+  description: "This post details the automated testing process for OpenSearch releases"
+---
+
+# Automated testing for OpenSearch releases
+
+OpenSearch releases many distributions across multiple platform flavors as part of a new version release. These distributions are fundamentally of two kinds - the full distribution that includes all OpenSearch plugins packaged with OpenSearch core, and the min distribution which is just the core without any plugins. These release distributions go through a rigorous testing process across multiple teams, before they are signed off as release ready. This includes validating unit test coverage, integration testing across all components to validate the behavior, backward compatibility testing to ensure upgrade compatibility with previous versions, and stress testing to validate the performance metrics. Once these distributions are successfully tested and hardened, they are marked ready for the release.
```

Comment:
- remove "fundamentally"
- we distribute OpenSearch min and a distribution with plugins, no need to mention "core", it's not really a thing
- quote "release ready"
- "hardened" implies some kind of security, I would remove